addons/radar/XEH_preInit.sqf

```diff
@@ -19,6 +19,12 @@ GVAR(setNamelist) = true;
 // compass whitelist, these need to be lowercase!!
 GVAR(compassWhitelist) = "getText (_x >> 'simulation') == 'ItemCompass'" configClasses (configFile >> "CfgWeapons") apply {configName _x};
 
+// GPS whitelist
+private _uavTerminals = "inheritsFrom _x == (configFile >> 'CfgWeapons' >> 'UavTerminal_base')" configClasses (configFile >> "CfgWeapons") apply {configName _x};
+private _gpsDevices = "getText (_x >> 'simulation') == 'ItemGPS'" configClasses (configFile >> "CfgWeapons") apply {configName _x};
```

Comment:
maybe combining this into one line would be smart

Reply:
Done